pkg/apis/sources/v1alpha1/kafka_types.go

```diff
@@ -121,6 +132,11 @@ type KafkaSourceStatus struct {
 	// * SinkURI - the current active sink URI that has been configured for the
 	//   Source.
 	duckv1.SourceStatus `json:",inline"`
+
+	// Total number of consumers targeted by this source
+	// +optional
+	// For round-tripping only.
+	Consumers int32 `json:"consumers,omitempty"`
```

Comment:
Why do we need this? Isn't the one in the spec enough?

Reply:
yes it's a required field for enabling the scale subresource. See https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource